docs/amber.md

```diff
@@ -0,0 +1,138 @@
+# Amber
+
+Amber is a multi-API shader test framework.Graphics bugs can be captured and
```

Comment:
I insist on at least one space after a sentence-terminating period.  (Normally I use 2 but I won't start a total religous war over it.)

Reply:
Done. I like one, you're wasting soooo many bytes by using 2. Won't you think of the spinning disks?